devpi_ldap/main.py

```diff
@@ -46,8 +47,15 @@ def __init__(self, path):
         with open(self.path) as f:
             _config = yaml.safe_load(f)
         self.update(_config.get('devpi-ldap', {}))
-        if 'url' not in self:
-            fatal("No url in LDAP config.")
+        if 'server_pool' not in self and 'url' not in self:
+            fatal("Neither pool nor url in LDAP config.")
```

Comment:
Here and in the docs you say "xor", yet if ``server_pool`` and ``url`` both exist the former silently overwrites the latter. This should be an error in my opinion, as it could be bad indentation or copy'n'paste in the config.

Reply:
Oh your are right, I should have at least put a warning for that case as i originally planned it but forgot to put it in.

But if you want to have this as an unrecoverable error, I don't mind. I can do that.

Reply:
Please make it an error, thanks!

Reply:
I will change it to a behavior that is in line with your call for the `server()` function. So either both get a warning (and a later removal) or both become unrecoverable errors.

Reply:
They are different things, removing the ``server`` function will eventually break existing things. The config change only breaks if a user changes the config to use the new ``server_pool`` functionality. So please leave ``server`` as is and make the config conflict an error.

Reply:
No problem. :)